velox/functions/lib/window/NthValue.cpp

```diff
@@ -305,7 +345,7 @@ void registerNthValue(const std::string& name, TypeKind offsetTypeKind) {
   exec::registerWindowFunction(
       name,
       std::move(signatures),
-      [name](
+      [name, offsetTypeKind](
```

Comment:
Does offsetTypeKind need to be a parameter ? You can obtain it in the constructor using the type of the second argument.

Reply:
Removed, thanks